### PR TITLE
Travis integration

### PR DESCRIPTION
**Pull request by sjwoodman**

### CODE REVIEW

File: .travis/build.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    docker build -t docker.io/streamziprocessors/cef-ops-random-data:latest random-data
+    docker push docker.io/streamziprocessors/cef-ops-random-data:latest
+
+    docker build -t docker.io/streamziprocessors/cef-ops-node-filter-event-data:latest nodejs/filter-events
+    docker push docker.io/streamziprocessors/cef-ops-node-filter-event-data:latest
+
+    docker build -t docker.io/streamziprocessors/cef-ops-receiver:latest amqp/receiver
+    docker push docker.io/streamziprocessors/cef-ops-receiver:latest
+
+    docker build -t docker.io/streamziprocessors/cef-ops-sender:latest amqp/receiver
```
Comment (matzew) [inline .travis/build.sh:25]: @sjwoodman let's include the new image from @hhiden in here as well

**Comment by matzew** [comment]
I've done some changes and merged a PR from @hhiden  - can you please rebase this PR, based on latest `master` branch ? 

Great work, seeing some sort of CI :tada:

**Comment by sjwoodman** [comment]
Rebased...

**Comment by matzew** [comment]
I think this is missing - sorry:

https://github.com/project-streamzi/cloudevent-flow/blob/master/pom.xml#L146-L152

**Comment by sjwoodman** [comment]
🎉 tests passing! :)

**Pull request merged**